doc/guide/Guide_CLIMADA_conventions.ipynb

```diff
@@ -223,7 +147,7 @@
     "\n",
     "| Availability | Name                                            | Version | Link | CLIMADA class | CLIMADA version | CLIMADA tutorial reference    |\n",
     "|--------------|-------------------------------------------------|---------|------|---------------|-----------------|-------------------------------|\n",
-    "| OK           | Fire Information for Resource Management System |         |[FIRMS](https://firms.modaps.eosdis.nasa.gov/download/) | BushFire      | >V1.2.5          | climada_hazard_BushFire.ipynb |\n",
+    "| OK           | Fire Information for Resource Management System |         |[FIRMS](https://firms.modaps.eosdis.nasa.gov/download/) | BushFire      | > v1.2.5          | climada_hazard_BushFire.ipynb |\n",
     "| OK           | Gridded Population of the World (GPW)           | v4.11   |[GPW4.11](https://sedac.ciesin.columbia.edu/data/set/gpw-v4-population-count-rev11)     | LitPop        | > v1.2.3        | climada_entity_LitPop.ipynb   |\n",
     "| FAILED       | Gridded Population of the World (GPW)           | v4.10   |[GPW1.10](https://sedac.ciesin.columbia.edu/data/set/gpw-v4-population-count-rev10)      | LitPop        | >= v1.2.0       | climada_entity_LitPop.ipynb   |\n",
```

Comment:
What does FAILED mean here? Can we simply remove the "Availability" column and the last row?

Reply:
I have no idea what it means, @chahank do you know? Would it be ok to remove the column and last row as @peanutfun suggests?